config/config.go

```diff
@@ -284,7 +284,16 @@ var Default = Config{
 	},
 }
 
-// writeDefault will create a default configuration file
+func WriteConf(path string, conf *Config) error {
+	bs, err := json.MarshalIndent(conf, "", "    ")
```

Comment:
New standards (since nodeJS) are to indent json files with 2 spaces.

Reply:
I would honestly go for tabs, but this is a war I don't want to get into. We use four spaces for JSONs everywhere (not just the gateway), so if we wanted to change it, it would need to be a consistent change. So not the time and place, I'm afraid.

Reply:
Yep - Was just an fyi rather than an issue.